nl.json

```diff
@@ -91,18 +91,102 @@
         },
         "shop" : {
             "title" : "Huidige winkel",
-            "title-featured" : "Aanbevolen",
+            "title-featured" : "Uitgelicht",
             "title-daily" : "Dagelijks",
-            "error" : "Winkel niet beschikbaar, probeer het later nog eens.",
-            "line1" : "De huidige item shop van Fortnite Battle Royale - dit wordt dagelijks bijgewerkt om [00:00 UTC](https://time.is/UTC|tooltip=Huidige tijd in UTC|newtab=true).",
-            "line2" : "Je kan de item shop van gisteren [hier](/shop/yesterday|tooltip=Item Shop van gisteren) zien.",
+            "error" : "Winkel niet beschikbaar, probeer het later opnieuw.",
+            "line1" : "De huidige item shop van Fortnite Battle Royale - word dagelijks bijgewerkt om [00:00 UTC](https://time.is/UTC|tooltip=Huidige tijd in UTC|newtab=true).",
+            "line2" : "Je kan de item shop van gisteren [hier](/shop/yesterday|tooltip=De Item Shop van gisteren) zien.",
```

Comment:
"[hier](/shop/yesterday|tooltip=De Item Shop van gisteren)"
Must be "[hier](/shop/yesterday|tooltip=Item Shop van gisteren)"

"De" can be removed!